### DISCUSSION TITLE
App does not work anymore after deployed to vercel

### DISCUSSION THREAD

**Discussion by felixarjuna** [vercel/next.js#54818] [Help]
My app works locally. After deploying to vercel, all of the api endpoints does not work anymore. I've been searching around and found that the common issue is that people hardcoded the server url. I don't think, this is the case for me. Thanks for anyone help! :)

Here is the error message i get:
```
{
  "error": {
    "message": "fetch failed",
    "code": -32603,
    "data": {
      "code": "INTERNAL_SERVER_ERROR",
      "httpStatus": 500,
      "path": "prayers.getPrayers"
    }
  }
}
```

I've check Vercel Logs too, the error message does not help me. I don't even think, there is a message there. 😀 
You can see them below. May be i miss something 

![image](https://github.com/vercel/next.js/assets/79026094/2b6adee8-066b-4513-8581-f157d094a3e6)

The stack i use for the backend:
`tRPC`, `Drizzle` and `PlanetScale`

Repository Link:
https://github.com/felixarjuna/PERKIWeb

**Comment by devjiwonchoi** [comment]
Not familiar with `@trpc` module, but is `endpoint: "/api/trpc",` a valid endpoint? Shouldn't it provide an actual URL?

```ts
import { appRouter } from "@/server";
import { fetchRequestHandler } from "@trpc/server/adapters/fetch";

const handler = async (req: Request) => {
  return await fetchRequestHandler({
    endpoint: "/api/trpc", // ----> here
    req,
    router: appRouter,
    createContext: () => ({}),
  });
};

export { handler as GET, handler as POST };
```

**Comment by metaYogeshBhutkar** [comment]
@felixarjuna did you find a solution for this?
